readme.md

```diff
@@ -205,7 +205,8 @@ The structure of a route event is: 'route:/path/to/resource:http-verb'. The rout
   res: response,
   req: request,
   params: the url parameters as an object,
-  query: the queryparams as an object
+  query: the queryparams as an object,
+  path: the pathParsed as an object
```

Comment:
do you adding the link to the path.parse documentation on node's documentation? That way users know what to expect in the object.

Reply:
of course.